### PR TITLE
Added file to guide: Javascript - standard objects - String fromCodePoint

### PR DESCRIPTION
**Pull request by kamatheuska**
Added file to guide: Javascript - standard objects - String fromCodePoint

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/javascript/standard-objects/string/string-fromcodepoint/index.md [hunk 1]
```diff
@@ -1,15 +1,49 @@
 ---
-title: String Fromcodepoint
+title: String FromCodePoint
 ---
-## String Fromcodepoint
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/standard-objects/string/string-fromcodepoint/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+`String.fromCodePoint` returns a string from a list or sequence of code points.
```
Comment (Nirajn2311) [inline guide/english/javascript/standard-objects/string/string-fromcodepoint/index.md]: Could you put a title for the article

File: guide/english/javascript/standard-objects/string/string-fromcodepoint/index.md [hunk 2]
```diff
@@ -1,15 +1,49 @@
...
+String.fromCodePoint(num1[, ...[, numN]])
+```
+
+### Parameters
+
+>**num1, ..., numN**
+
+A sequence of code points.
+
+## Description
```
Comment (Nirajn2311) [inline guide/english/javascript/standard-objects/string/string-fromcodepoint/index.md]: Also I think that it is better if the description was before the syntax

**Review by Nirajn2311 — CHANGES_REQUESTED** [review]

File: guide/english/javascript/standard-objects/string/string-fromcodepoint/index.md (hunk 1 above)
Comment (kamatheuska) [reply guide/english/javascript/standard-objects/string/string-fromcodepoint/index.md]: Sure! I though it had a title already.

File: guide/english/javascript/standard-objects/string/string-fromcodepoint/index.md (hunk 2 above)
Comment (kamatheuska) [reply guide/english/javascript/standard-objects/string/string-fromcodepoint/index.md]: Done.

**Review by Nirajn2311 — APPROVED** [review]

**Pull request merged**